src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -57,7 +57,7 @@
         networks                  (rf/sub [:wallet/network-details])
         token                     (rf/sub [:wallet/wallet-send-token])
         loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
-        token-symbol              (:symbol token)
+        token-symbol              (or (:symbol token) :eth)
```

Comment:
are we sure we want to default to eth? could this lead to some confusion ever? Is there a "safer" alternative if so?

Reply:
Not sure, but it should be imperceptible, because as soon as `:wallet/wallet-send-token` sub updates, `token-symbol` should have the correct value, which should happen really fast.

Reply:
@briansztamfater @J-Son89 I think the reason for this to break is something different.


This `token-symbol` is in the "on mount" part of the component, so when props change, the code isn't re executed:
```clojure
(defn component [props]
  (let [,,,
        token-symbol (or (:symbol token) :eth) ;;<- here
        ,,,]
    (fn [props]
      [rn/view ,,,]))) ;; <- instead of here
```

Also something not common is many of the involved  subs are in the "on mount" part :thinking:  It's  weird because the error is given by `make-limit-label`, if we follow the dependency chain:

`make-limit-label` -> `@current-limit` -> `token-symbol` (the var that now has a falback) -> `(rf/sub [:wallet/wallet-send-token])`

we see that the sub is declared to be only executed once, so I think the value won't be updated to have the correct one :thinking: 

Anynway, about a cleaner solution, I think about two options:

1. making the sub ` (rf/sub [:wallet/wallet-send-token])` return fallback values when the DB is empty.

2. (easier) Make the function that is breaking (`make-limit-label`) able to work with `nil` values:
```clojure
(defn- make-limit-label
  [{:keys [amount currency]}]
  (str amount " " (string/upper-case (name currency)))) ;; an option is to use `some->`
```
so we get:
```clojure
(defn- make-limit-label
  [{:keys [amount currency]}]
  (str amount " " (some-> currency name string/upper-case)))
```

But we should notice this is the same problem we have been having in other PRs about taking a token symbol and transform it to a user-friendly name, and we should do it in a more general way.

BTW another thing is we rename`token-symbol` as  `currency` in this component, that could lead to confusion, I think we usually call `currency` to fiat related vars (e.g. USD).

Reply:
@ulisesmac I changed the approach to yours :) LMK if that's okay so we can start testing this PR as is blocking others